charts/images.yaml

```diff
@@ -46,9 +46,9 @@ images:
   tag: "v1.3.0"
   targetVersion: ">= 1.22"
 - name: ingress-default-backend
-  sourceRepository: https://github.com/kubernetes/ingress-gce
-  repository: k8s.gcr.io/defaultbackend-amd64
-  tag: "1.5"
+  sourceRepository: github.com/gardener/ingress-default-backend
+  repository: eu.gcr.io/gardener-project/gardener/ingress-default-backend
+  tag: "0.11.0"
```

Comment:
The livenessProbe of the Deployment has to be adapted.
Did you tested this change before opening PR?

Reply:
Guestbook test passes without adapting health endpoint also but now I checked this https://github.com/gardener/gardener/pull/6006 and have adapted the changes accordingly.